### PR TITLE
Add ddprocmon versions handling in release.create-rc task

### PR DESCRIPTION
**Pull request by kacper-murzyn**
### What does this PR do?

windows-kernel-integration team added recently new entries to the `release.json` file. They were not considered when running `in release.create-rc` task which we use to generate new RC entries and ended up being discarded. This PR adds missing functionality.

### Motivation

Automation of Agent RC build process.

### Describe how to test/QA your changes

Tested locally in mocked scenario. This needs to be properly tested when creating next real RC build.

### CODE REVIEW

File: tasks/release.py [hunk 1]
```diff
@@ -558,7 +560,25 @@ def _get_windows_ddnpm_release_json_info(release_json, agent_major_version, is_f
...
     print(f"The windows ddnpm version is {win_ddnpm_version}")
 
-    return win_ddnpm_driver, win_ddnpm_version, win_ddnpm_shasum
+    # Step 2: DDPROCMON
+
+    win_ddprocmon_driver = release_json_version_data['WINDOWS_DDPROCMON_DRIVER']
+    win_ddprocmon_version = release_json_version_data['WINDOWS_DDPROCMON_VERSION']
+    win_ddprocmon_shasum = release_json_version_data['WINDOWS_DDPROCMON_SHASUM']
+
+    if win_ddnpm_driver not in ['release-signed', 'attestation-signed']:
```
Comment (chouetz) [inline tasks/release.py:569]:
```suggestion
    if win_ddprocmon_driver not in ['release-signed', 'attestation-signed']:
```

File: tasks/release.py [hunk 2]
```diff
@@ -558,7 +560,25 @@ def _get_windows_ddnpm_release_json_info(release_json, agent_major_version, is_f
...
+        print(f"WARN: WINDOWS_DDPROCMON_DRIVER value '{win_ddprocmon_driver}' is not valid")
+
+    print(f"The windows ddprocmon version is {win_ddnpm_version}")
```
Comment (chouetz) [inline tasks/release.py:572]:
```suggestion
    print(f"The windows ddprocmon version is {win_ddprocmon_version}")
```

File: tasks/release.py [hunk 3]
```diff
@@ -558,7 +560,25 @@ def _get_windows_ddnpm_release_json_info(release_json, agent_major_version, is_f
 
     print(f"The windows ddnpm version is {win_ddnpm_version}")
 
-    return win_ddnpm_driver, win_ddnpm_version, win_ddnpm_shasum
+    # Step 2: DDPROCMON
```
Comment (chouetz) [inline tasks/release.py:563]: Not necessarily easier but a small method could have helped as it's mostly duplication of the above

**Review by chouetz — CHANGES_REQUESTED** [review]
Small typos

File: tasks/release.py (hunk 3 above)
Comment (kacper-murzyn) [reply tasks/release.py]: I guess that makes sense. Refactored.

File: tasks/release.py (hunk 2 above)
Comment (kacper-murzyn) [reply tasks/release.py]: Fixed

File: tasks/release.py (hunk 1 above)
Comment (kacper-murzyn) [reply tasks/release.py]: Fixed

**Review by chouetz — APPROVED** [review]

**Review by derekwbrown — APPROVED** [review]

**Pull request merged**